### ISSUE TITLE
[Bug] Option [port,timeout] must be an integer

### ISSUE THREAD

**Issue by rognales** [DirectoryTree/LdapRecord#308]
**Environment (please complete the following information):**
 - LDAP Server Type: ActiveDirectory
 - PHP Version: 7.4.8
 - Laravel Version:  8

Just bump this base package just now and received below error. ````php artisan ldap:test```` also no longer works

````
Upgrading directorytree/ldaprecord (v2.5.0 => v2.5.1)
````

In summary, any defaults config that is integer, will failed this validation since .env is normally parsed as string

https://github.com/DirectoryTree/LdapRecord/blob/24c2fdd9707a10a266d25e98d100a4f5865c581b/src/Configuration/DomainConfiguration.php#L168

 ````Option [port] must be an integer.````
 ````Option [timeout] must be an integer.````

**Comment by rognales** [comment]
This is my config

````
LDAP_LOGGING=true
LDAP_CONNECTION=default
LDAP_HOST=1.2.3.4
LDAP_USERNAME="cn=SECRET,ou=serviceAccount,o=Cortana"
LDAP_PASSWORD=SUPERSECRET
LDAP_PORT=636
LDAP_BASE_DN="ou=users, o=data"
LDAP_TIMEOUT=2
LDAP_SSL=true
LDAP_TLS=true
````

**Comment by rognales** [comment]
I did logging the value passed to the Configurator

````
[2021-06-24 14:29:15] local.INFO: ConfigurationException ["hosts",["1.2.3.4"]] 
[2021-06-24 14:29:15] local.INFO: ConfigurationException ["username","cn=SECRET,ou=serviceAccount,o=Cortana"] 
[2021-06-24 14:29:15] local.INFO: ConfigurationException ["password","SUPERSECRET"] 
[2021-06-24 14:29:15] local.INFO: ConfigurationException ["port","636"] 
````

**Comment by milanchheda-cactus** [comment]
I am setting this up on a fresh Laravel and getting the exact same error.

`Option [port] must be an integer.`

**Comment by rognales** [comment]
> All variables in your .env files are typically parsed as strings

[Environment variable type](https://laravel.com/docs/8.x/configuration#environment-variable-types)

**Comment by milanchheda-cactus** [comment]
Found a work-around here: https://github.com/DirectoryTree/LdapRecord-Laravel/issues/308#issuecomment-867424525

**Comment by stevebauman** [comment]
Sorry about that everyone! I just patched this in [v2.5.2](https://github.com/DirectoryTree/LdapRecord/releases/tag/v2.5.2). I added a test to make sure this doesn't happen again.
